labs/multiverse/src/widgets/room_view/mod.rs

```diff
@@ -620,6 +645,9 @@ impl Widget for &mut RoomView {
             let maybe_room = self.client.get_room(room_id);
             let mut maybe_room = maybe_room.as_ref();
 
+            let items = self.get_selected_timeline_items();
+            let selected_event = self.get_selected_event();
```

Comment:
Consider pushing these two lines down just before (current) line 671, so they're closer to their use (and not unused if the mode is normal)?